src/Tribe/Event_Cleaner_Scheduler.php

```diff
@@ -223,10 +223,27 @@ public function move_old_events_to_trash():array {
 			return $results;
 		}
 
+		remove_action( 'save_post_' . Tribe__Events__Main::POSTTYPE, [
```

Comment:
I'd _suggest_ moving this (and the re-add) to separate functions so if we find we have to add more remove/add(s) it doesn't clutter this function up so much (it's already the majority of the code in here)